# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: b53fd89c3789
# Full Hash: b53fd89c3789279bff2f41c5e2061a1ffd915625
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Use kCodecParamCbr as intended. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D128384
# ==============================================================================

diff -r fa792634c1de -r b53fd89c3789 dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Fri Oct 08 16:10:41 2021 -0700
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Thu Oct 14 08:24:55 2021 +0000
@@ -815,7 +815,7 @@
       parameters[kCodecParamMaxPTime] = std::to_string(aConfig.mMaxFrameSizeMs);
     }
     if (aConfig.mCbrEnabled) {
-      parameters["cbr"] = kParamValueTrue;
+      parameters[kCodecParamCbr] = kParamValueTrue;
     }
   }
 
